server/data-import/data.js

```diff
@@ -25,23 +22,36 @@ loadData()
 
 // note that exports are 'mongoexport --db hah-dev --collection pregnancycenters --jsonArray --out cessilye_nypc_geocoded.json'
 
-PregnancyCenterModel.collection.dropAsync()
-	.catch( (err) => log.error(err))
-	.then( () => {
-		fs.readFile('test/fixtures/cessilye_nypc.json', 'utf8', (err, data) => {
-			const docs = EJSON.parse(data)
-			log.info(typeof docs)
-			for (const doc of docs) {
+async function reimport() {
 
-				Joi.validateAsync(doc, pregnancyCenterSchemaJoi, {abortEarly: false})
-					.catch( (err) => log.error(err))
-					.then( (validatedData) => {
-						PregnancyCenterModel.createAsync(validatedData)
-							.then( (pc) => log.info(pc))
-					})
+	let docs
 
-			}
+	try {
+		await PregnancyCenterModel.collection.drop()
+	} catch (err) {
+		log.error(err)
+	}
 
-		})
+	try {
+		const dataToImport = fs.readFileSync('server/data-import/test/fixtures/cessilye_nypc.json', 'utf8')
+		docs = EJSON.parse(dataToImport)
+	} catch (err) {
+		log.error(err)
+	}
 
-	})
+	for (const doc of docs) {
+		const validationObj = await Joi.validate(doc, pregnancyCenterSchemaJoi, {
+			abortEarly: false
+		})
+		if (validationObj.error) {
+			log.error(validationObj.error)
```

Comment:
Just out of curiosity, what does the `Joi` object look like if it does fail validation? I'm thinking we may want to pull out some key pieces to make debugging easier (and it may be something we want to use for all validation). Also, is it an asynchronous call? Do you need to use `await`?